components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -81,12 +80,16 @@
 import { compose, propOr, head } from 'ramda'
 
 import TagList from '@/components/TagList/TagList.vue'
+
+import NotifySuccess from '@/mixins/notify/index'
+import NotifyFail from '@/mixins/notify/index'
 export default {
   name: 'DatasetAboutInfo',
 
   components: {
     TagList
   },
+  mixins: [NotifySuccess, NotifyFail],
```

Comment:
By default, everything defined in a mixin is merged into the component it is imported to. This means all methods, computed, data, etc. will be part of the component that you merged into. This means you only have to import the notify mixin, and define it once. 

[Mixin docs for more info](https://vuejs.org/v2/guide/mixins.html)
```suggestion

import NotifyMixin from '@/mixins/notify/index'

export default {
  name: 'DatasetAboutInfo',

  components: {
    TagList
  },
  mixins: [NotifyMixin],
```

Reply:
see my comment about mixins...

![image](https://user-images.githubusercontent.com/7054394/85593253-df4a2180-b614-11ea-8fef-427302b2cc8c.png)